src/vm/eventpipefile.h

```diff
@@ -153,6 +154,8 @@ class EventPipeFile final : public FastSerializableObject
     // The frequency of the timestamps used for this file.
     LARGE_INTEGER m_timeStampFrequency;
 
+    StreamWriter *m_pStreamWriter;
```

Comment:
I would make this a `const` pointer, initialized on the ctor., and add `PRECONDITION(pStreamWriter != nullptr)`